### PR TITLE
graphviz install experimentation

### PR DESCRIPTION
**Pull request by i13302**
it cat print version of graphziv.

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run -it --rm --volume `PWD`/work:/home/jovyan/work/ superml /bin/bash
```
Comment (gacky35) [inline start.sh]: I've fixed it like 
`docker run -it --rm -p 8888:8888 --volume $(pwd)/work:/home/jovyan/work/ superml /bin/bash`
and I checked that it works.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM jupyter/base-notebook
+
+USER root
+RUN apt-get update -y && apt-get install -y --no-install-recommends build-essential graphviz
+
```
Comment (gacky35) [inline Dockerfile:5]: If we want to execute hello.py from [this page](https://graphviz.readthedocs.io/en/stable/examples.html).
We have to install xdg-open.
So, I add this line here. `RUN apt-get install -y xdg-utils --fix-missing`
And, I checked hello.py works.

File: start.sh (hunk 1 above)
Comment (i13302) [reply start.sh]: soory

File: Dockerfile (hunk 2 above)
Comment (i13302) [reply Dockerfile:5]: xdg-open is package for GUI (x11).
i would rather not  install it. 

may be it used `g.view()` in hello.py.

**Comment by i13302** [comment]
Done is better than perfect ! by Zuckerberg, Facebook.

File: start.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run -it --rm --publish 8888:8888 --volume `PWD`/work:/home/jovyan/work/ superml
```
Comment (gacky35) [inline start.sh]: This doesn't work.
I have to change `PWD` to $(pwd)

**Pull request merged**